content/docs/12-enterprise-version/Helm Chart Install References

```diff
@@ -0,0 +1,161 @@
+| `insecureSkipVerifyTIs` | `True` | Boolean | Specifies whether to skip TLS verification for the SMTP connection.| Optional |
+| `fromEmailID` | `noreply@spectrocloud.com` | String |SEmail ID used from From address.| Optional |
+| `password` | `YmFzZTY0X2VtYWlsX3Bhc3N3b3Jk` | String |Base64-encoded SMTP password when sending emails.| Optional |
+
+
+### Environment parameters
+
+The below table outlines the **values.yaml** parameters for configuring a **MongoDB** deployment. 
+
+| **Parameters** | **Default value** | **Type** | **Description** | **Required/Optional** |
+| --- | --- | --- | --- | --- |
+| `env.rootDomain` | "" | String | Specifies the default DNS record mapped to the ingress-nginx-controller load balancer. It is mandatory if false; otherwise, leave it empty. | Required|
+| `env.installerMode` | `self-hosted` | String | Installer mode specified, do not modify the value.| Required|
+| `env.installerCloud` | "" | String | Cloud platform unspecified, leave empty. | Optional |
+
+
+
+### Cluster parameters
+
+Cluster parameters are used to specify the Kubernetes cluster configuration.
+
+
+| **Parameters** | **Default value** | **Type** | **Description** | **Required/Optional** |
+| --- | --- | --- | --- | --- |
+| `stableEndpointAccess` | `False` | Boolean | Specifies whether EKS cluster is deployed in private network access through a stable private endpoint, if true, through a public endpoint. | Optional|
+
+## Registry and SCAR parameters
+
+**Registry and SCAR (Spectro Cloud Admission Registrar)** parameters are mandatory parameters in the **values.yaml** for installing **Palette** using Helm Chart.
+
+| **Parameters**       | **Type**   | **Description**                                             | **Required/Optional** |
+| ----------------- | ------ | -------------------------------------------------------- | ------------------ |
+| `registry.endpoint` | String | The endpoint URL for the registry.                      | Required           |
+| `registry.name`     | String | The name of the registry.                                | Required           |
+| `registry.password` | String | The password for the registry.                           | Required           |
+| `registry.username` | String | The username for the registry.                           | Required           |
+| `scar.endpoint`     | String | The endpoint URL of the Spectro Cloud Admission Registrar (SCAR). | Required           |
+| `scar.username`     | String | The username for the SCAR.                               | Required           |
+| `scar.password`     | String | The password for the SCAR.                               | Required           |
+
+To learn about the default values of the parameters, please contact Spectro Cloud sales.
+
+## Network Address Translation(NATS) parameters
+
+This section explains the parameters for the configuration file containing settings related to the **Network Address Translation (NATS)** messaging system. 
```

Comment:
This language is a bit different from what has been used before. Keep it the same as the other intro sentences.